examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -30,9 +31,21 @@ using Oceananigans.Units: minute, minutes, hour
 # ## The grid
 #
 # We use 32³ grid points with 2 m grid spacing in the horizontal and
-# 1 m spacing in the vertical,
+# varying spacing in the vertical, with higher resolution closer to the
+# surface,
 
-grid = RegularRectilinearGrid(size=(32, 32, 32), extent=(64, 64, 32))
+σ = 1.1 # stretching factor
+Nz = 24
+Lz = 32
+
+hyperbolically_spaced_faces(k) = - Lz * (1 + (tanh(σ * ( (1-k+Nz) / Nz - 1)) / tanh(σ)))
+
+grid = VerticallyStretchedRectilinearGrid(size = (32, 32, Nz), 
+                                          x = (0, 64),
+                                          y = (0, 64),
+                                          z_faces = hyperbolically_spaced_faces)
+
+plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz], marker=:circle, ylabel="Depth (m)", xlabel="Vertical spacing (m)", legend=nothing)
```

Comment:
There is more than one way to define a hyperbolically stretched grid, though. I think plotting is a good idea also because we can discuss and make sure this is the type we want

Reply:
I think a comment about why we are creating a plot and what it shows is a good idea to include, in addition to the plot itself.

Reply:
Yes. I wanna inspect the plot first before making any additional changes, so I'll wait for the preview to build

Reply:
Plots will appear if they are the last line in a code block (eg if there is literate'd text following the plot). Literate works (I believe) by calling `show` on the output of the last line in the code block. So the output is identical to calling `pl = plot(...); show(pl)`. This is how I recommend previewing results. It's painful to build docs for this purpose in my experience.

We make a simple plot like this in the Kelvin-Hemlholtz example:

https://clima.github.io/OceananigansDocumentation/stable/generated/kelvin_helmholtz_instability/

The animation is also an example.

Reply:
Everything seems fine plotting it that way. So I guess we're good